root/main/ConfirmSeed.js

```diff
@@ -29,6 +29,17 @@ const ConfirmSeed = ({
 }: Props): React$Element<typeof Layout> => {
   const $c = React.useContext(SanitizedCatalystContext);
   const title = l('Confirm Form Submission');
+
+  /*
+   * Automatically submit the form if /release/add (where POST requests are
+   * used to seed fields rather than to create edits) was requested and the
+   * client indicated that they want the form to be skipped. See MBS-13225.
+   */
+  const url = new URL($c.req.uri);
+  const autoSubmit =
+    url.pathname === '/release/add' &&
+    url.searchParams.get('skip_confirmation') === '1';
```

Comment:
I used a body parameter for this initially, but it resulted in an "unknown parameter" error further down the line from the release form. Let me know if it seems like it'd be better to use a body parameter here but then strip it out so it doesn't get re-posted.

Reply:
(resolving this since it sounds from another comment like it's fine to use a query param)